doc/src/base/base.md

```diff
@@ -355,6 +355,10 @@ Base.@allocations
 Base.EnvDict
 Base.ENV
 Base.Sys.STDLIB
+Base.Sys.CPU_NAME
+Base.Sys.JIT
+Base.Sys.cpu_info
+Base.Sys.cpu_summary
```

Comment:
We shouldn't need to document this function anymore in the the manual, as it is obsoleted by the `show` method.
```suggestion
```